src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -801,7 +805,7 @@ void uct_rc_mlx5_txqp_dptr_post_iov(uct_rc_iface_t *iface, enum ibv_qp_type qp_t
     uct_rc_mlx5_common_post_send(iface, qp_type, txqp, txwq,
                                  opcode_flags & UCT_RC_MLX5_OPCODE_MASK,
                                  0, fm_ce_se, wqe_size, av, grh_av, ib_imm_be,
-                                 max_log_sge);
+                                 max_log_sge, NULL);
```

Comment:
can we say log_sge is always NULL for inline case? so remove the parameter log_sge from uct_rc_mlx5_txqp_inline_post?

Reply:
wanted to unify these calls...
ok, removed